bundles/org.openhab.ui.basic/web-src/smarthome.js

```diff
@@ -1491,6 +1492,40 @@
 		_t.input.addEventListener("change", onChange);
 	}
 
+	/* class ControlInput extends Control */
+	function ControlInput(parentNode) {
+		Control.call(this, parentNode);
+
+		var
+			_t = this;
+
+		_t.input = _t.parentNode.querySelector("input[type=text]");
+
+		_t.valueNode = _t.parentNode.parentNode.querySelector(o.formValue);
```

Comment:
I should normally find "mdl-form__value" in your snippet but I don't.

Reply:
I have a specific class input__value for the value. I didn't get the expected result combining mdl-form__value with mdl-textfield__input, so created input, input__value and input__label to complement mdl-textfield. The main issue was with alignment.

Reply:
But `o.formValue` is ".mdl-form__value".  I do not understand how `_t.valueNode` is properly initialized.

Reply:
You are right. I changed to a specific class, but didn't change this. It didn't work anymore.